rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,292 @@
+                },
+                "value": "1064-3745",
+                "type": "Identifier"
+            }
+        ],
+        "title": "Methods in Molecular Biology, Methods and Protocols",
+        "items": [
+            {
+                "title": "212",
+                "type": "Item",
+                "locations": []
+            }
+        ],
+        "type": "Work"
+    },
+    {   // 773 
+        "title": "Springer eBooks",
+        "title": "Work"
+    }
+]
+```
+- [3001878](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3001878?lang=eng&suite=cobalt&marcData=Y)
+  Only 830 with no 490/440 or 773
+```
+"partOf": [
+    {
+        "title": "Adamson Collection Wellcome Library", // subfield $a concatenated with $p
+        "type": "Work"
+    }
+]
+```
+- [1110225](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1110225?lang=eng&suite=cobalt&marcData=Y)
+Serie statement in 440 with only the title
+```
+"partOf": [
+    {
+        "title": "Monographs on inorganic and physical chemistry",
+        "type": "Work"
+    }
+]
+```
+- [1204561](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1204561?lang=eng&suite=cobalt&marcData=Y) 440 with id
+```
+"partOf": [
+    {
+        "id": "abcdefgh",
+        "identifiers": [
+            {
+                "identifierType": {
+                    "id": "issn",
+                    "label": "ISSN",
+                    "type": "IdentifierType"
+                },
+                "value": "0920-8607",
+                "type": "Identifier"
+            }
+        ],
+        "title": "Brill's studies in intellectual history",
+        "items": [
+            {
+                "title": "v. 13",
+                "type": "Item",
+                "locations": []
+            }
+        ],
+        "type": "Work"
+    }
+]
+```
+- [2301867](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2301867?lang=eng&suite=cobalt&marcData=Y) 
+  773 not overlapping with series and no id. 
+```
+"partOf": [
+    {
+        "title": "Eighteenth Century collections online",
+        "type": "Work"
+    }
+]
+```
+- [1186777](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1186777?lang=eng&suite=cobalt&marcData=Y) 
+  773 with an id in subfield $w which links to another Work in our library (see example below) 
+```
+"partOf": [
+    {
+        "id": "abcdefgh",
+        "identifiers": [
+            {
+                // identifier from $w
+                "identifierType": {
+                    "id": "???",
+                    "label": "???",
+                    "type": "IdentifierType"
+                },
+                "value": "(Wcat)9175i",
+                "type": "Identifier"
+            }
+        ],
+        "title": "Basil Hood. Photograph album",
+        "items": [
+            {
+                "title": "page 9", 
+                "type": "Item",
+                "locations": []
+            }
+        ]
+    }
+]
+```
```

Comment:
Comments from @jtweed:
> - This is a link to something that exists in the catalogue: https://wellcomecollection.org/works/tw5yfsfv
> - I'll be honest, my inital thought was it was a ID from WorlCat, but it's not, it's one of our iconographic numbers
> - These are unhelpfully currently in shelfmarks, but there is a piece of work for Alex to do soon that moves them to referenceNumber
> - We should probably add them as identifiers too when we do that work
> - That would mean we could search in $w against the identifiers that exist on works, assuming it's a id scheme we recognise
> - I don't know how consistent the use of Wcat is, I would have expected UkLW, so that's a question for #collections-info
> - Given that, in this instance the linked work should be embedded as per (and possibly by) the relation embedder